packages/nextjs/hooks/scaffold-eth/useOutsideClick.ts

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from "react";
 
 /**
- * Check if a click was made outside the passed ref
+ * Checks if a click was made outside of passed ref
```

Comment:
It's not only check, main thing here is handling click outside (callback).

So I think it should be something like `Handles clicks outside of passed ref`

Reply:
Oh yes, thanks! fixed at [4b737b5](https://github.com/scaffold-eth/scaffold-eth-2/pull/665/commits/4b737b594e65216499a5cc4158ac3d72ad5cc0db)